xiaomi_gateway/__init__.py

```diff
@@ -328,7 +323,7 @@ def push_data(self, data):
             return False
         sid = data['sid']
         for func in self.callbacks[sid]:
-            func(jdata)
+            func(jdata, data)
```

Comment:
I do not like that the `jdata`  now is  send twice

Reply:
I don't think it's bad. Semantically it's like "data" and "raw response for extra cases".
We can send `cmd` instead, but in future we  may need other fields of a raw response to be accessible in devices and interface will be changed again.

Reply:
Ok, I will merge this when the HA pr is ready